packages/web-pkg/src/services/uppy/uppyService.ts

```diff
@@ -307,4 +308,15 @@ export class UppyService {
       item.value = null
     })
   }
+
+  /**
+   * Set a specific upload folder for an upload. The HandleUpload plugin
+   * checks, if a specific folder has been specified as upload destination.
+   * If not, it falls back to the current folder.
+   * The uploadId needs to be set within the meta object of the upload files
+   * for the plugin to connect an upload to its destination folder.
+   **/
+  setUploadFolder(uploadId: string, folder: Resource) {
+    this.uploadFolderMap[uploadId] = folder
```

Comment:
How can this be removed from the map again?

Reply:
Currently not 😅 I guess it would be a little cleaner...